newdle/api.py

```diff
@@ -356,9 +365,32 @@ def update_participant(args, code, participant_code):
                 },
             )
         participant.newdle.update_lastmod()
+        formatted_answers = [
+            'Date: {} ({})'.format(
+                date.strftime('%d/%m/%Y, %H:%M:%S'), states_titles.get(state)
```

Comment:
```suggestion
                date.strftime('%d/%m/%Y, %H:%M:%S'), states_titles[state]
```

you never have a state that's not in the titles dict